<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1530</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btn1">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>580</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>ENDE</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>425</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>800</width>
      <height>425</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>800</width>
      <height>425</height>
     </size>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_1">
     <attribute name="title">
      <string>Panel 1</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Panel 2</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Panel 3</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Panel 4</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_5">
     <attribute name="title">
      <string>Panel 5</string>
     </attribute>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>0</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>I/F PANEL SETUP</string>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>30</y>
      <width>551</width>
      <height>391</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="6" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>TEXT COLOR</string>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>INTER LOCK ENABLE</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QComboBox" name="cbSecurity"/>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>SECURITY</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>ARRANGE</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>PANEL TYPE</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>GROUP NAME</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QComboBox" name="cbType"/>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="cbARrange">
       <item>
        <property name="text">
         <string> select ARrange</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1A</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1B</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1C</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1D</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1E</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1F</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1G</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1H</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2A</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2B</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2C</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2D</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2E</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2F</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2G</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2H</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>3A</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>3B</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>3C</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>3D</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>3E</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>3F</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>3G</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>3H</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>4A</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>4B</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>4C</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>4D</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>4E</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>4F</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>4G</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>4H</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QComboBox" name="cbType1"/>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>INPUT (DISP)</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QPlainTextEdit" name="name1">
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>25</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>PANEL COLOR</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>PANEL NAME</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>SETUP</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QComboBox" name="cbTextColor"/>
     </item>
     <item row="4" column="1">
      <widget class="QComboBox" name="cbColor"/>
     </item>
     <item row="9" column="1">
      <widget class="QComboBox" name="cbInput"/>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="cbSetup"/>
     </item>
     <item row="8" column="1">
      <widget class="QComboBox" name="cbInterlock"/>
     </item>
     <item row="5" column="2">
      <widget class="QPlainTextEdit" name="name2">
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>25</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="5" column="3">
      <widget class="QPlainTextEdit" name="name3">
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>25</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPlainTextEdit" name="groupName">
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>25</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1530</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDatei">
    <property name="title">
     <string>Datei</string>
    </property>
    <addaction name="actionLoad"/>
    <addaction name="actionSave"/>
   </widget>
   <addaction name="menuDatei"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionLoad">
   <property name="text">
    <string>Load</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
